src/bincode/shortvec.zig

```diff
@@ -6,7 +6,8 @@ const bincode = sig.bincode;
 const serialize_short_u16 = sig.bincode.varint.serialize_short_u16;
 const deserialize_short_u16 = sig.bincode.varint.deserialize_short_u16;
 
-pub fn ShortVecConfig(comptime Child: type) bincode.FieldConfig([]Child) {
+pub fn sliceConfig(comptime Slice: type) bincode.FieldConfig(Slice) {
```

Comment:
if we dont support anything other than slices of a child, not sure why we want to make them define a slice type instead of defining the child ?

Reply:
It allows specifying more types of slices - namely immutable slices. This is taken advantage of in `core/transaction.zig`.